src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebar.tsx

```diff
@@ -74,6 +77,9 @@ export const ArtworkSidebar: React.FC<ArtworkSidebarProps> = ({
   })
 
   const artworkEcommerceAvailable = !!(isAcquireable || isOfferable)
+  const artsyAuction = partner?.internalID === "6321be2a8cde97000d9ad2df"
```

Comment:
@Mitch-Henson - mind adding a comment here about whats going on? The reasoning behind this hard-coded ID will be lost to time quicker than not.

Reply:
Delayed reply but updated [here](https://github.com/artsy/force/pull/14746)